upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -521,6 +521,27 @@ func (b *BootstrapChannelBuilder) buildAddons() *channelsapi.Addons {
 		}
 	}
 
+	if b.Cluster.Spec.ClusterAutoscaler != nil && fi.BoolValue(b.Cluster.Spec.ClusterAutoscaler.Enabled) {
```

Comment:
Perhaps we should default `Enabled` to true if `ClusterAutoscaler` is non-`nil`

Reply:
We agreed to keep it disabled by default at last office hours.
Maybe in 1.20 we materialise it into to the cluster spec so that people know it's enabled.

Reply:
But default is to have a `nil` `ClusterAutoscaler` field. Why would someone specify a `ClusterAutoscaler` field yet want `Enabled` to default to `false`?

Reply:
I agree that it may be a good idea in general, just don't know how used this pattern is in kops.
Sometimes when I test or just want to disable something I just comment the `enabled: true` line, without actually but keep all other setting to make it easy to reenable.

One of the reasons for starting the discussion on https://github.com/kubernetes/kops/issues/9661 was to establish some guidelines for enabled/disabled and managed/unmanaged, but seems not very popular topic. Maybe should open a more generic one.